src/js/services/profileService.js

```diff
@@ -433,6 +433,39 @@ angular.module('copayApp.services')
       });
     };
 
+    root.checkLatestRelease = function(cb) {
+      var releaseURL = configService.getDefaults().release.url;
+
+      root.requestLatestRelease(releaseURL, function(err, release) {
+        if (err) return cb(err);
+
+        var actualVersion = window.version;
+        var latestVersion = release.data.tag_name;
+        var majorOfActualVersion = actualVersion.split('.')[0];
```

Comment:
names should be: currentVersion and latestVerstion.  currentVersionMajor, and currentVersionMinor